### PR TITLE
feat: adding border to google buttons on light mode

### PR DESCRIPTION
**Pull request by TomDowling**
## Changes
Adding a border so the google button is more visible on light mode. Screenshots added below

dark-mode
light-mode

### Describe what this PR does
- Short and concise, bullet points can help
- Screenshots if applicable can also help

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1560 #done

### CODE REVIEW

File: packages/shared/src/components/auth/AuthDefault.tsx [hunk 1]
```diff
@@ -160,7 +160,7 @@ const AuthDefault = ({
             key={provider}
             provider={provider}
             label={shouldLogin ? 'Log in with' : 'Sign up with'}
-            className="mb-1"
+            className="mb-1 border-0"
```
Comment (rebelchris) [inline packages/shared/src/components/auth/AuthDefault.tsx:163]: Does this not overwrite the className in the providers map though? 👀

**Review by rebelchris — COMMENTED** [review]
Small concern, just not sure on how it renders all together now.

File: packages/shared/src/components/auth/AuthDefault.tsx (hunk 1 above)
Comment (TomDowling) [reply packages/shared/src/components/auth/AuthDefault.tsx:163]: Yes it does i was leaving this pages setup as it was, but i could remove this and deal with the spacing (margin) with gap or space-y which would be better?

Comment (rebelchris) [reply packages/shared/src/components/auth/AuthDefault.tsx:163]: I mean more for this it won't take the border color then right? 
As we never merge the -1 border now.

File: packages/shared/src/components/auth/ProviderButton.tsx [hunk 2]
```diff
@@ -24,7 +24,6 @@ function ProviderButton({
       style={{
         justifyContent: label ? 'flex-start' : 'center',
         color: '#FFFFFF',
-        border: 0,
```
Comment (sshanzel) [inline packages/shared/src/components/auth/ProviderButton.tsx:27]: I think the reason we had this as an inline style was to override the button's base class for border. Though if the `className` can do that no issues, then we can move forward with this.

File: packages/shared/src/components/auth/common.tsx [hunk 3]
```diff
@@ -41,6 +41,7 @@ export const providerMap: ProviderMap = {
     icon: <GoogleIcon className="socialIcon" secondary />,
     provider: 'Google',
     style: { backgroundColor: '#FFFFFF', color: '#0E1217' },
+    className: 'border border-theme-divider-tertiary',
```
Comment (sshanzel) [inline packages/shared/src/components/auth/common.tsx:44]: Is this expected for both light and dark?

Comment (TomDowling) [reply packages/shared/src/components/auth/common.tsx:44]: From the slack thread i read it as they wanted this color (divider-teritary) for both

Comment (rebelchris) [reply packages/shared/src/components/auth/common.tsx:44]: Yep, should be fine indeed, it sorts light/dark itself

File: packages/shared/src/components/auth/AuthDefault.tsx (hunk 1 above)
Comment (TomDowling) [reply packages/shared/src/components/auth/AuthDefault.tsx:163]: This doesn't get used on the google button as we are passing classNames in the providers map, so overrides this classNames to the one supplied in the map as its passed in { ...props } at the bottom of the props. I have confirmed it works currently as can be seen in the screenshots above :)

Comment (TomDowling) [reply packages/shared/src/components/auth/AuthDefault.tsx:163]: I have refactored this now so will resubmit for review in a few

**Review by rebelchris — APPROVED** [review]
Great!

Let's go 🎉

**Pull request merged**